arches/app/etl_modules/base_import_module.py

```diff
@@ -158,40 +175,66 @@ def get_node_lookup(self, nodes):
         return lookup
 
     def run_load_task(self, userid, files, summary, result, temp_dir, loadid):
-        with connection.cursor() as cursor:
```

Comment:
I factored out stage_files() and check_tile_cardinality(), and then wrapped what remained in a try/finally so that temp files were always cleaned up.